pkg/sql/catalog/lease/lease.go

```diff
@@ -610,9 +625,22 @@ func acquireNodeLease(
 
 // releaseLease from store.
 func releaseLease(ctx context.Context, lease *storedLease, m *Manager) {
+	releaseLeaseWithNotify(ctx, lease, m, nil)
+}
+
+// releaseLeaseWithNotify from store and notifies a channel.
```

Comment:
Is "notify" a standard term for closing a channel? notify, to me, reads as "Sends a value".